libs/qmlglsink/CMakeLists.txt

```diff
@@ -87,11 +87,17 @@ if(QGC_ENABLE_VIDEOSTREAMING)
         endif()
 	endif()
 
-	find_package(PkgConfig)
-	if(NOT EXISTS ${GSTREAMER_ROOT})
-		pkg_check_modules(GSTREAMER gstreamer-1.0>=${GST_TARGET_VERSION})
-		if(GSTREAMER_FOUND)
-			set(GSTREAMER_ROOT ${GSTREAMER_PREFIX})
+	if(MACOS)
+		find_library(GSTREAMER_FRAMEWORK
+			NAMES GStreamer
+			HINTS  ${GSTREAMER_ROOT})
```

Comment:
Because we are using a Framework for gstreamer you use find_library to find it. cmake supports finding frameworks in find_library.